manifests/h/Harmonoid/Harmonoid/0.3.5.0/Harmonoid.Harmonoid.installer.yaml

```diff
@@ -0,0 +1,22 @@
+# Created with YamlCreate.ps1 v2.0.7 using InputObject 🤖 $debug=QUSU-7-2-3
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: Harmonoid.Harmonoid
+PackageVersion: 0.3.5.0
+InstallerType: inno
+Scope: machine
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+UpgradeBehavior: install
+Dependencies:
+  PackageDependencies:
+  - PackageIdentifier: Microsoft.VCRedist.2015+.x64
+ReleaseDate: 2022-11-16
+Installers:
+- Architecture: x64
+  InstallerUrl: https://github.com/alexmercerind2/harmonoid-releases/releases/download/v0.3.5/harmonoid-windows-setup.exe
+  InstallerSha256: B9B8D153F581460DFA8D0AFB4DEF2EEC6E60ECE6CC0D499FC5713E37D0975D57
```

Comment:
```suggestion
  InstallerSha256: B9B8D153F581460DFA8D0AFB4DEF2EEC6E60ECE6CC0D499FC5713E37D0975D57
    InstallerSwitches:
      Custom: /allusers
```

Reply:
The installer prompts for installation location - in Program Files or in the user's AppData Local folder. This switch installs it into the recommended location.
![image](https://user-images.githubusercontent.com/114952565/210853603-6dfe04ad-b65a-45be-8734-e7906ef2b4e7.png)